src/lib.rs

```diff
@@ -333,6 +383,18 @@ impl Pixels {
         Ok(())
     }
 
+    // Re-create the swap chain with its own values
+    pub(crate) fn re_create_swap_chain(&mut self) {
```

Comment:
nitpick: minor name change to `recreate_swap_chain` would be preferred.

Reply:
Of course, done.